lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -277,7 +292,15 @@
     'storage_class_name': {'type': 'str'},
     'volume_mode': {'type': 'str'},
     'volume_name': {'type': 'str'},
-    'cdi_source': {'type': 'dict'}
+    'cdi_source': {'type': 'dict'},
+    'wait': {
+        'type': 'bool',
+        'default': False
```

Comment:
What's the reason to have this false by default?

Reply:
A valid cluster configuration is one that uses `volumeBindingMode: WaitForFirstConsumer`, which in practice means that defaulting to `True` would result in timeouts on perfectly valid configs. Hence the need to default to `False`.